lib/cli/index.js

```diff
@@ -96,6 +96,8 @@ var _ = require('lodash'),
             help: 'DEPRECATED: Run a single folder from a collection. To be used with -c or -u'
         });
 
+        // @todo This is conflicting with '-r: --reporters'. Should we remove this?
+        // Also for this we have '--timeout-request'
         parser.addArgument(['-r', '--requestTimeout'], {
```

Comment:
This set of arguments is only handled when `run` is not specified, the diff can be removed.